moveit_ros/visualization/rviz_plugin_render_tools/src/trajectory_visualization.cpp

```diff
@@ -120,7 +120,7 @@ TrajectoryVisualization::~TrajectoryVisualization()
 }
 
 void TrajectoryVisualization::onInitialize(Ogre::SceneNode* scene_node, rviz_common::DisplayContext* context,
-                                           const rclcpp::Node& ros_node)
+                                           rclcpp::Node::SharedPtr ros_node)
```

Comment:
Rever back the const

Reply:
0fc4f6629d8e7ffadaa13200233f1c483bb1f241